aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +258,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form
```

Comment:
I'm a little confused how this is being presented. Wasn't the original issue to answer the question of "when should I use a form in a side layer vs. in a full page experience?"

I would expect the title of this section to be "Form presentation" or "Forms in Layers vs. dedicated page" or "Forms in Layer vs. full-page"...

Reply:
You are correct. The initial question was that but up on talking to the designers who had this questions and use cases they were in need of a guidance on how to use full page since we had a forms in layer guidance.  I do like the comparison section name."Forms in Layer vs. full-page" . Do you suggest i should link the forms in layer with in this guidance ?

Reply:
Yes, cross-linking would be great.